doc/visual-programming/source/widgets/model/svm.rst

```diff
@@ -0,0 +1,83 @@
+SVM
+===
+
+.. figure:: icons/svm.png
+
+Support Vector Machines map inputs to higher-dimensional feature spaces.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+   A data set.
+
+-  **Preprocessor**
+
+   Preprocessing method(s)
+
+**Outputs**:
+
+-  **Learner**
+
+   A support vector machine learning algorithm with settings as specified in the dialog.
+
+-  **Model**
+
+   A trained model. Output signal sent only if input *Data* is present.
+
+-  **Support Vectors**
+
+   A subset of data instances from the training set that were used as support vectors in the trained model.
+
+Description
+-----------
+
+`Support vector machine <https://en.wikipedia.org/wiki/Support_vector_machine>`_ (SVM) is a machine learning technique that separates the attribute space with a hyperplane, thus maximizing the margin between the instances of different classes or class values. The technique often yields supreme predictive performance results. Orange embeds a popular implementation of SVM from the `LIBSVM <https://www.csie.ntu.edu.tw/~cjlin/libsvm/>`_ package. This widget is its graphical user interface.
+
+For regression tasks, **SVM** performs linear regression in a high dimension feature space using an ε-intensive loss. Its estimation accuracy depends on a good setting of C, ε and kernel parameters. The widget outputs class predictions based on a `SVM Regression <https://en.wikipedia.org/wiki/Support_vector_machine#Regression>`_.
+
+The widget works for both classification and regression tasks.
+
+.. figure:: images/SVM-stamped.png
+   :scale: 50 %
+
+1. The learner can be given a name under which it will appear in other widgets. The default name is "SVM".
```

Comment:
The description seems fine, but I would like to point out something that could potentially be confusing.

With the regular SVM, there are two settings
1. **C** is just the penalty term for the loss and applies everywhere.
2. **epsilon** is a parameter to the epsilon-SVR model, and only applies to regression tasks (unfortunately, since the learner can't know what type of data it will receive in advance, it has to be enabled all the time. I've tried to make it clear that it will only be applied to regression tasks with *Regression* in the name.

The nu-SVM is similar:
1. **C** only applies to regression tasks here and is the penalty of the error.
2. **nu** applies to both problems.